### PR TITLE
docs: add upgrade docs after migration

### PR DESCRIPTION
**Pull request by wenytang-ms**

### CODE REVIEW

File: packages/fx-core/resource/upgrade-change-logs.md [hunk 1]
```diff
@@ -18,5 +18,11 @@ Teams Toolkit now supports provision Azure resources using an Infrastructure as
 ### Environment Specific Configuration Files
 Teams Toolkit now supports creating multiple environments for a project, and you can customize the configurations for a specific environment, for example using different app names in the manifest for different environments. You can read more about what you can do in [this wiki](https://github.com/OfficeDev/TeamsFx/wiki/Enable-Preview-Features-in-Teams-Toolkit#managing-multiple-environments-in-teams-toolkit).
 
+## Know what you need to do
+### Provision resource with bot
+If you have already provisioned the bot service before the migration, and you want to continue to use the bot service after the migration, please provision again. We will create a new bot service for this project, and other resources will not change.
+
+### Use exsiting resource instead of provision new resources
```
Comment (MuyangAmigo) [inline packages/fx-core/resource/upgrade-change-logs.md:25]: Typo: `the guid` -> `the guide`?

File: packages/fx-core/resource/upgrade-change-logs.md [hunk 2]
```diff
@@ -18,5 +18,11 @@ Teams Toolkit now supports provision Azure resources using an Infrastructure as
 ### Environment Specific Configuration Files
 Teams Toolkit now supports creating multiple environments for a project, and you can customize the configurations for a specific environment, for example using different app names in the manifest for different environments. You can read more about what you can do in [this wiki](https://github.com/OfficeDev/TeamsFx/wiki/Enable-Preview-Features-in-Teams-Toolkit#managing-multiple-environments-in-teams-toolkit).
 
+## Know what you need to do
+### Provision resource with bot
+If you have already provisioned the bot service before the migration, and you want to continue to use the bot service after the migration, please provision again. We will create a new bot service for this project, and other resources will not change.
```
Comment (MuyangAmigo) [inline packages/fx-core/resource/upgrade-change-logs.md:23]: Can we explain why we will provision a new bot service, in a separate doc/

File: packages/fx-core/resource/upgrade-change-logs.md (hunk 1 above)
Comment (wenytang-ms) [reply packages/fx-core/resource/upgrade-change-logs.md:25]: updated

File: packages/fx-core/resource/upgrade-change-logs.md (hunk 2 above)
Comment (wenytang-ms) [reply packages/fx-core/resource/upgrade-change-logs.md]: good idea!

**Review by MuyangAmigo — APPROVED** [review]

**Pull request merged**